com.microsoft.mrtk.input/MRTK.Input.asmdef

```diff
@@ -6,7 +6,10 @@
         "Microsoft.MixedReality.Toolkit.Core",
         "Unity.InputSystem",
         "Unity.XR.CoreUtils",
-        "Unity.XR.Interaction.Toolkit"
+        "Unity.XR.Interaction.Toolkit",
+        "Unity.XR.OpenXR",
```

Comment:
Adding OpenXR as a dep here is probably fine; might cause problems if we ever try to run MRTK3 on non-OpenXR platforms (Magic Leap 1, ML2 (for now))

Reply:
What's this dependency used for? on a quick scroll through here, I didn't see the namepace used, but I might have missed it.

Reply:
Fixed and removed